src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -581,44 +602,37 @@ static term nif_gpio_digital_read(Context *ctx, int argc, term argv[])
     return gpio_digital_read(argv[0]);
 }
 
-static const struct Nif gpio_set_pin_mode_nif =
-{
+static const struct Nif gpio_set_pin_mode_nif = {
```

Comment:
Was this an editor doing auto-indent?  We should probably restore this to the convention of curly braces on new lines for struct declarations.